plugins/py3embed/py3embed.cpp

```diff
@@ -44,6 +44,11 @@
 #include "enginecontext.hpp"
 #include <regex>
 
+#if defined (__linux__) || defined(__FreeBSD__)  || defined(__APPLE__)
+#include <execinfo.h> // comment out if not present
```

Comment:
info: Gordon pointed me to a new c++17 feature:

#if __has_include(<span>)

To check if an include file exists - that might be useful here.